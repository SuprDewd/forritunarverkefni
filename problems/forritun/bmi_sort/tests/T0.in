3

John
1.5
60

Jane
1.4
55

Bob
2.0
95
