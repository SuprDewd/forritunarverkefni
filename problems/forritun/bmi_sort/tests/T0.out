Bob 23.8
John 26.7
Jane 28.1
